contracts/src/v0.8/dev/interfaces/OCR2DRRegistryInterface.sol

```diff
@@ -83,4 +83,13 @@ interface OCR2DRRegistryInterface {
     uint256 reportValidationGas,
     uint256 initialGas
   ) external returns (bool success);
+
+  /**
+   * @notice Gets subscription owner.
+   * @param subscriptionId - ID of the subscription
+   * @return owner - owner of the subscription.
+   */
+  function getSubscriptionOwner(
```

Comment:
What if we made this fetch the whole subscription instead? In case we ever need to get something off of it later

Reply:
Do you think this is worth adding now or can we make this change later if needed?

I would advocate for not worrying about this now as the OracleRequest event may change more prior to mainnet per our discussion yesterday.

Reply:
Sounds good to me. I'll add a new Jira ticket so that we remember.